etc/config/c.defaults.properties

```diff
@@ -1,5 +1,5 @@
 # Default settings for C
-compilers=&gcc:&clang
+compilers=&gcc:&clang:&cerberus
```

Comment:
Hi!

I don't think this is of general enough use to make one of the default compilers. Here we only have GCC and Clang as they're expected to be installed on most systems (or at least one of them).

I don't think we should add cerberus to the defaults at all.